mvc-login/src/main/resources/templates/fragments/navbar.html

```diff
@@ -53,7 +53,7 @@
                             <a href="/profile" class="dropdown-item dropdown-profile">
                                 <i class="fas fa-user mr-3"></i> Profile
                             </a>
-                            <a id="qsLogoutBtn" th:href="@{/logout}" class="dropdown-item">
+                            <a id="qsLogoutBtn" href="javascript: document.logoutForm.submit()" class="dropdown-item">
```

Comment:
I cringe when I see inline `javascript:` for links. Is there a better way to do this?

Reply:
I'm not a fan either; previously we just used a link perform logout, but that made a GET request to `/logout` instead of a POST. I'll look for a better way.

Reply:
I think we can do something like this instead of using an anchor tag:

```html
<form name="logoutForm" th:action="@{/logout}" method="post">
    <button type="submit" class="dropdown-item">
        <i class="fas fa-power-off mr-3"></i> Log out
    </button>
</form>
```